tests/gold_tests/tls/tls_client_cert.test.py

```diff
@@ -171,6 +173,20 @@
 tr2.Processes.Default.Env = ts.Env
 tr2.Processes.Default.ReturnCode = 0
 
+# Parking this as a ready tester on a meaningless process
```

Comment:
I wonder if we should make this kinda a autest standard function? eg, seems like other tests might want to have a check to see if xxx.config/yaml has completed?